besspin/cyberPhys/cyberphyslib/cyberphyslib/demonstrator/director.py

```diff
@@ -214,6 +226,8 @@ def component_error_send(self, component_id, error_id):
                              dlc=canlib.CAN_DLC_CMD_COMPONENT_ERROR,
                              data=struct.pack(canlib.CAN_FORMAT_CMD_COMPONENT_ERROR,
                                               component_id, error_id))
+    def status_send(self, canid, argument):
+        msg = extcan.Message(arbitration_id=canid, dlc=1, data=struct.pack("!B", argument))
```

Comment:
I usually do `from can import Message` so I would prefer to keep it consistent

Reply:
Few more things:
1. `cc_recvr` -> `cmd_bus` (for consistency)
2. `status_send` seems to be just a thin wrapper around two lines of code - perhaps not necessary?
3. please use `canlib.CAN_FORMAT...` and `canlib.CAN_DLC_...` to specify format/DLC - that way we don't accidentally make mistakes

Reply:
We will work on the director after this merge